site/sass/components/_banner-interstitial.scss

```diff
@@ -0,0 +1,102 @@
+@use 'sass:math';
+
+.banner-interstitial {
+  $self: &;
+  $break-2-col: 603px;
+  $scale-max: 1600px;
+  $scale-range: ($break-2-col $scale-max);
+  $clamp-check: clamp(1px, 1vw, 2px);
+  $begin-color-stop: var(--primary-blue), var(--primary-blue) math.percentage(math.div(211, 696)), var(--secondary-blue) math.percentage(math.div(211, 696));
+  $gradient-angle-l: 105deg;
+
+  display: grid;
+  position: relative;
+
+  @include mq($break-2-col) {
+    background-image: linear-gradient($gradient-angle-l, $begin-color-stop, var(--secondary-blue) calc(math.percentage(math.div(211, 696)) + 188px), var(--primary-blue) calc(math.percentage(math.div(211, 696)) + 188px), var(--primary-blue));
+    border-radius: 0.625rem;
+
+    // scale the width of the color stop if supported.
+    @supports (width: min(1px, 1vw)) {
+      background-image: linear-gradient($gradient-angle-l, $begin-color-stop, var(--secondary-blue) calc(math.percentage(math.div(211, 696)) + min(20%, 234px)), var(--primary-blue) calc(math.percentage(math.div(211, 696)) + min(20%, 234px)), var(--primary-blue));
+    }
+  }
+
+  &__wrapper {
+    @include mq(0 $break-2-col - 1px) {
+      background-image: linear-gradient(65deg, var(--secondary-blue), var(--secondary-blue) 115px, var(--primary-blue) 115px, var(--primary-blue));
+      overflow: hidden;
+    }
+  }
+
+  &__content {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(pxRem(268px), 1fr));
+    position: relative;
+  }
+
+  &__text {
+    $block-spacing-range: (24px 96px);
+
+    padding-top: 3rem;
+    z-index: 1;
+
+    @include mq($break-2-col) {
+      padding: 3.5rem 0 3.5rem 3rem;
+      place-self: center flex-start;
+
+      @supports (padding: $clamp-check) {
+        padding: generateScaling($scale-range, $block-spacing-range) 0 generateScaling($scale-range, $block-spacing-range) generateScaling($scale-range, (40px 124px));
+      }
+    }
+  }
+
+  &__eyebrow,
+  &__title,
+  &__copy {
+    color: var(--white);
+  }
+
+  &__copy {
+    margin-top: 1.25rem;
+    max-width: 42ch;
+
+    #{$self}--no-cta & {
+      margin-bottom: 6rem;
+    }
+  }
+
+  &__cta {
+    display: inline-flex;
+    margin-top: 2.5rem;
+  }
+
+  &__image-wrapper {
+    position: relative;
+    width: math.percentage(math.div(456, 327));
```

Comment:
Out of curiosity, where are these numbers coming from? Can you include a comment explaining it? And if they're, say, image dimensions, please include the `px` as Sass should do the unit math for us